cirq/google/engine/engine.py

```diff
@@ -185,15 +185,15 @@ def __init__(self,
             discoveryServiceUrl=discovery_service_url,
             **kwargs)
 
-    def run(self,
+    def run(
+            self,
             *,  # Force keyword args.
             program: Union[circuits.Circuit, Schedule],
             job_config: Optional[JobConfig] = None,
             param_resolver: ParamResolver = ParamResolver({}),
             repetitions: int = 1,
             priority: int = 50,
-            target_route: str = '/xmonsim'
-            ) -> TrialResult:
+            processor_ids: List[str] = ['xmonsim']) -> TrialResult:
```

Comment:
Use an immutable tuple instead of a mutable list for the default value. https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

The type `Sequence[str]` matches both.

Reply:
Done.